optimum/exporters/onnx/utils.py

```diff
@@ -366,6 +370,7 @@ def get_stable_diffusion_models_for_export(
         onnx_config_constructor = TasksManager.get_exporter_config_constructor(
             model=pipeline.text_encoder_2,
             exporter="onnx",
+            library_name="diffusers",
```

Comment:
same for text encoder 2: https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0/blob/main/model_index.json

Reply:
same